RELEASING.adoc

```diff
@@ -1,80 +1,210 @@
 = Releasing Kiali
 
-This document gives the steps required to perform a release of Kiali. This document is only useful to Kiali release managers.
+This document covers the specifics of releasing Kiali, and is meant as a guide
+for Kiali release managers.
+
+The summary of the process is the following:
+
+* Create a new JIRA issue for the release
+* Make sure Getting Started docs are complete
+* Update Kiali UI version
+* Publish the API Docs
+* Update Makefile
+* Create the Release PR
+* Tag Git Repo
+* Notify Istio and Openshift Istio Teams for changes:
+** Istio Helm Charts
+** Istio Ansible Installer
+** Openshift Istio Ansible Installer (istiooc)
+* Update JIRA issues included with the release
+* Send out notifications (twitter, mailing lists, IRC)
 
-The steps to perform a release are relatively simple thanks to the way link:.travis.yml[Travis] is setup.
+== Creating a new JIRA issue
 
-Before running these steps, it is assumed the link:https://github.com/kiali/kiali/tree/master[kiali master] branch and the link:https://github.com/kiali/kiali-ui/tree/master[kiali-ui master] branch are all up to date and include all code that is to be released and that your local `master` branches are up to date with the remote `master` branches.
+Create a new issue on link:https://issues.jboss.org/browse/KIALI[JIRA] detailing the
+release, and add sub-tasks if necessary.
 
-For the steps below, it is assumed the image version being released is v1.0.0. Obviously, this version string for the docker image will change as new images are released.
+== Updating Docs
 
-== Kiali-UI
+First step is to make sure the "Getting Started" document for the website is up
+to date and works with the new version.
 
-1) Tag the Kiali-UI master branch in this repo: https://github.com/kiali/kiali-ui
+The docs can be found link:https://github.com/kiali/kiali.io[here]. Do the
+changes, and create a PR.
 
-NOTE: You must have write permissions to the public GitHub repository in order to be able to do this.
+== Updating Kiali UI
 
-For example,
+The specifics of releasing Kiali UI can be found
+link:https://github.com/kiali/kiali-ui/blob/master/RELEASING.adoc[here].
 
-```
-$ git tag v1.0.0
-$ git push origin v1.0.0
-```
+Releasing Kiali UI should come before releasing the core, as the image is
```

Comment:
You should release Kiali UI before releasing Kiali core, as Travis will create the new image as soon as the tag for core is pushed to master.